### PR TITLE
Bug 1140613 - [Contacts] The star indicating a favorite contact overlaps long contact names in the header of the contact details page r=francisco

### PR DESCRIPTION
**Pull request by julienw**

### CODE REVIEW

File: shared/elements/gaia-header/dist/gaia-header.js [hunk 1]
```diff
@@ -140,23 +140,12 @@ return w[n];},m.exports,m);w[n]=m.exports;};})('font-fit',this));
  */
```
Comment (julienw) [inline shared/elements/gaia-header/dist/gaia-header.js]: This will be updated in a separate commit once I got a r+ for https://github.com/gaia-components/gaia-header/pull/27

I include it here to make it easier to test.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7146c08263ba78c95f28f6a95583f9d45a95dcd9

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=14aecb1b1ab9e2979ab4d1f4e37a0ab1524a00f1
